Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -376,6 +369,54 @@ CGImageRef CGImageCreateWithMaskingColors(CGImageRef image, const CGFloat* compo
 
 #pragma region WIC_HELPERS
 
+DisplayTexture* _CGImageGetDisplayTexture(CGImageRef image) {
+    RETURN_NULL_IF(!image);
+    CGIWICBitmap* iwicImg = dynamic_cast<CGIWICBitmap*>(image->ImageSource().Get());
+    RETURN_NULL_IF(!iwicImg);
+    return iwicImg->GetDisplayTexture()->GetTexture();
+}
+
+// Return the data pointer to the Image data.
+void* _CGImageObtainImageBytes(CGImageRef image) {
+    RETURN_NULL_IF(!image);
+    return image->Data();
+}
+
+CGImageRef _CGImageCreateCustomWICImage(IDisplayTexture* texture, WICPixelFormatGUID pixelFormat, UINT height, UINT width) {
+    RETURN_NULL_IF(!texture);
+
+    ComPtr<IWICBitmap> bitmap = Make<CGIWICBitmap>(texture, pixelFormat, height, width);
+    RETURN_NULL_IF(!bitmap);
+
+    CGImageRef imageRef = __CGImage::CreateInstance();
+    imageRef->SetImageSource(bitmap);
+
+    return imageRef;
+}
+
+CGImageRef _CGImageConvertImageToPixelFormat(CGImageRef image, WICPixelFormatGUID pixelFormat) {
```

Comment:
>_CGImageConvertImageToPixelFormat [](start = 11, length = 33)

nit: Rework this name so it has `Create` in it; Create says "I am returning a new +1 object"